packages/smooth_app/lib/pages/preferences/user_preferences_connect.dart

```diff
@@ -108,7 +109,28 @@ class UserPreferencesConnect extends AbstractUserPreferences {
               attachmentPaths: includeLogs == true ? Logs.logFilesPaths : null,
             );
 
-            await FlutterEmailSender.send(email);
+            try {
+              await FlutterEmailSender.send(email);
+            } on PlatformException catch (e) {
+              if (e.code == 'not_available') {
```

Comment:
Is this code the same on android & iOS

Reply:
Yes https://github.com/sidlatau/flutter_email_sender/blob/master/android/src/main/kotlin/com/sidlatau/flutteremailsender/FlutterEmailSenderPlugin.kt#L179

Reply:
Perfect